sphinx/tutorial/rose/suites.rst

```diff
@@ -239,63 +229,63 @@ Using a Rose workflow configuration with Cylc 8
 
       Open the :ref:`command-rose-config-edit` GUI. You should see
       :guilabel:`suite conf` in the panel on the left-hand side of the window.
-      This will contain the environment and template variables we have just
-      defined.
+      This will contain the template variables we have just defined.
 
-   #. **Use Suite Variables In The** ``flow.cylc`` **File.**
+   #. **Use suite variables in the** ``flow.cylc`` **file**
```

Comment:
This heading was de-capitalised for some reason, but others in the same file were not?